TEI.js

```diff
@@ -59,17 +59,27 @@
 // Zotero.addOption("exportNotes", false);
 // Zotero.addOption("generateXMLIds", true);
 
-var ns = {
+const ns = {
 	tei: "http://www.tei-c.org/ns/1.0",
 	xml: "http://www.w3.org/XML/1998/namespace"
 };
 
 
-var exportedXMLIds = {};
-var generatedItems = {};
-var allItems = {};
+const exportedXMLIds = {};
+const generatedItems = {};
+const allItems = {};
+
+
+// build one time
+const xmlparser = new DOMParser();
+const xmlser = new XMLSerializer();
 
-// replace formatting with TEI tags
+/**
+ * Replace formatting with TEI tags.
+ * [2023-06 FG] Legacy, @see appendXML(), to have formatting as element node
+ * @param {*} title 
+ * @returns 
+ */
 function replaceFormatting(title) {
```

Comment:
If you're replacing something, you should replace it — not just leave behind an unused copy with duplicate logic.

Reply:
I thought it was easier for reviewers to compare with the old code, it is of course deleted on my branch, quite far from that now. This little PR is a first contact with you, to learn your good practices.

Reply:
This will be done for next submission, if I can solve your point upper.